e2e/testcases/helm_sync_test.go

```diff
@@ -380,3 +409,75 @@ func helmChartVersion(chartVersion string) nomostest.Sha1Func {
 		return chartVersion, nil
 	}
 }
+
+// pushes a new helm chart for use during an e2e test
+func pushHelmChart(nt *nomostest.NT, t *testing.T, helmchart, version string) string {
```

Comment:
This looks like it should be a utility in `nomostest`. Also should return an error so that the caller can decide what to do with the error

Reply:
done